src/ResourceManager/HDInsight/Commands.HDInsight.Test/UnitTests/ConfigurationTests.cs

```diff
@@ -41,6 +42,43 @@ public void CanCreateNewConfigForRServer()
             CreateNewConfig(setEdgeNodeVmSize: true);
         }
 
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void CanAddSparkCustomConfigs()
+        {
+            AzureHDInsightConfig config = new AzureHDInsightConfig();
+
+            var addConfigValuesCmdlet = new AddAzureHDInsightConfigValuesCommand
```

Comment:
@jeffersonezra should this test be updated since it is using Spark and Spark2 in the same cmdlet call? Or at least have this test split into two, one for each version?